Engine/fgens.c

```diff
@@ -1755,7 +1730,7 @@ static int gen30(FGDATA *ff, FUNC *ftp)
     for (i = 0; i < l2; i++)
       x[i] = xsr * f2[i];
     /* filter */
-    csound->RealFFT(csound, x, l2);
+    csoundRealFFT(csound, x, l2);
```

Comment:
Not a blocker, but I thought we had moved to using csound->APIFunc() in opcodes and fgens to have parity with external opcodes, and used csoundAPIFunc() for core code and host code. I may be recalling this incorrectly though. Is there a particular reason to change to use csoundRealFFT here instead of csound->RealFFT? (@vlazzarini)